view.go

```diff
@@ -26,15 +26,22 @@ func (r Reference) Summarize(style lipgloss.Style) string {
 
 // Describe creates a full, formated description of a reference
 func (r Reference) Describe(style lipgloss.Style) string {
+	defer func() {
+		if rec := recover(); rec != nil {
+			fmt.Fprintf(os.Stderr, "An error occurred: %v\n", rec)
+		}
+	}()
+
 	statusBar := renderStatusBar(r.Name, r.Summary)
 	descriptionStyle, err := updateTermRendered(style)
 	if err != nil {
-		// hoping this doesn't happen as most commands here suceed without issue
-		panic(err)
+		fmt.Fprintf(os.Stderr, "Failed to update term renderer: %v\n", err)
+		return "Error rendering description"
 	}
 	description, err := descriptionStyle.Render(r.Description)
 	if err != nil {
-		panic(err)
+		fmt.Fprintf(os.Stderr, "Failed to render description: %v\n", err)
+		return "Error rendering description"
```

Comment:
I think I would prefer returning `(string, error)` here. Then we can handle the failure by exiting with a non-zero code properly.